templates/pricing/pro.html

```diff
@@ -561,40 +652,34 @@ <h3 class="p-heading--2">Ceph and Swift</h3>
             </tr>
           </tbody>
         </table>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-6">&nbsp;</div>
-      <div class="col-6">
         <p>
           Prices are for Ubuntu Pro + 24/7 support contracts.
           <br class="u-hide--small" />
           Weekday support is discounted by 50%.
         </p>
-        <p>
+        <div class="p-cta-block">
           <a href="/support/contact-us" class="p-button--positive js-invoke-modal">Get in touch</a>
-        </p>
+        </div>
       </div>
     </div>
   </section>
 
-  <section class="p-strip is-deep u-no-padding--top">
-    <hr class="p-rule is-fixed-width" />
-    <div class="row">
-      <div class="col-6">
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
         <h3 class="p-heading--2">Fully managed OpenStack, Kubernetes and LMA</h3>
         <p>
           We manage infrastructure at both the host and the guest level. Infrastructure includes Ceph and Swift storage, Kubernetes, OpenStack and the recommended logging, monitoring and alerting stack.
         </p>
       </div>
-
-      <div class="col-6">
-        <table class="p-table is-not-indented p-table--openstack-k9s-lma">
+      <div class="col">
+        <table class="p-table is-not-indented">
           <thead>
             <tr>
-              <th width="35%">Feature</th>
-              <th class="u-align--right">Per VM</th>
-              <th class="u-align--right">Per host</th>
+              <th width="35%" scope="col">Feature</th>
+              <th class="u-align--right" scope="col">Per VM</th>
+              <th class="u-align--right" scope="col">Per host</th>
             </tr>
           </thead>
           <tbody>
```

Comment:
It won't let me comment on the line directly, but on 691 on the asterisk we can add an aria label (aria-desribedby? I think that'd be the most appropriate attribute but could be wrong) to link it to the "On a VMware or OpenStack cluster..." text

Reply:
I was trying to get aria-describedby to work, as it seems the most appopriate but it turns out it has some different behaviours depending on whether it is used in a form or not. There is an interesting [stackoverflow post I was reading](https://stackoverflow.com/questions/56465819/accessibility-in-what-scenarios-would-aria-describedby-not-be-announced). In this scenario I had to use an aria-label and just explicitly add the footnote. Again, not sure if this is the 'correct' method, but it works